eppo/src/main/java/cloud/eppo/android/util/Utils.java

```diff
@@ -54,8 +54,14 @@ public static String getISODate(Date date) {
         return dateFormat.format(date);
     }
 
-    public static SharedPreferences getSharedPrefs(Context context) {
-        return context.getSharedPreferences("eppo", Context.MODE_PRIVATE);
+    public static String safeCacheKey(String key) {
+        // Take the first eight characters to avoid the key being sensitive information
+        // Remove non-alphanumeric characters so it plays nice with filesystem
+        return key.substring(0,8).replaceAll("\\W", "");
```

Comment:
Ah yeah this is confusing `keySuffix` refers to us adding a suffix onto the cache key (filename) but we use the prefix of the _API_ key. Since we're removing shared preferences I think I can just rename this to the more descriptive `cacheFileNameSuffix`